tests/test_client.py

```diff
@@ -97,7 +97,7 @@ def test_add_auth_headers(self, simple_config):
         authed = client._add_auth_headers(prepared)
         assert prepared.headers != authed.headers
         assert authed.headers["X-API-Hash"] == "71373f0a57118b120d1915ccc0a24ae2cc112ad3"
-        assert authed.headers["X-API-Microtime"] == b"1582761600.0"
+        assert authed.headers["X-API-Microtime"] == "1582761600.0"
```

Comment:
is this...okay? Are we sure this works? I'm pretty sure I wrote this to behave the same as the perl client, so this just worries me a bit.

Reply:
I am honestly not sure, @okurz do you have an idea? The issue is that requests' typing complains that `bytes` are not supposed to be in the header. But that could be wrong of course.

Reply:
I don't know. But can you exclude this from the typing check somehow?